src/components/todo-list/todo-list.jsx

```diff
@@ -0,0 +1,112 @@
+import styled from "./todo-list.module.scss";
+import Image from "next/image";
+import useSearchTasks from "./useSearchTasks";
+import { useState, useRef } from "react";
+
+const TodoList = (tasks) => {
+  console.log(tasks);
+  const taskTitleRefs = useRef([]);
+  const [currentTasks, setCurrentTasks] = useState(tasks.props);
+  const [query, setQuery] = useState("");
+
+  const handleClick = (e, index) => {
+    const taskTitleRef = taskTitleRefs.current[index];
+    taskTitleRef.isCompleted = !taskTitleRef.isCompleted;
+    if (taskTitleRef && taskTitleRef.isCompleted === true) {
+      taskTitleRef.style.textDecoration = "line-through";
+      taskTitleRef.style.color = "rgba(22, 31, 62, 0.3)";
+    }
+    if (taskTitleRef && taskTitleRef.isCompleted === false) {
```

Comment:
`taskTitleRef.isCompleted` уже является логической переменной. Жесткое сравнение необязательно
```suggestion
    if (taskTitleRef && taskTitleRef.isCompleted) {
      taskTitleRef.style.textDecoration = "line-through";
      taskTitleRef.style.color = "rgba(22, 31, 62, 0.3)";
    }
    if (taskTitleRef && !taskTitleRef.isCompleted) {
```